src/routes/+page.svelte

```diff
@@ -75,26 +135,66 @@
 		{:else if arrivalsAndDepartures.length > 0}
 			<div class="flex flex-col divide-y divide-gray-300">
 				{#each arrivalsAndDepartures as dep (dep.tripId)}
-					<div class="flex items-center gap-x-4 p-4">
-						<div class="rounded-lg bg-gray-800 p-4 text-2xl font-bold text-white">
-							{dep.routeShortName}
-						</div>
-						<div class="flex-1 text-xl">
-							{dep.tripHeadsign}
-						</div>
-						<div class="text-right">
-							{#if isComingSoon(dep.predictedDepartureTime, dep.scheduledDepartureTime)}
-								<div class="text-5xl font-bold">
-									{getArrivalStatus(dep.predictedDepartureTime, dep.scheduledDepartureTime)}
-								</div>
-								<div class="text-sm">min</div>
-							{:else}
-								<div class="text-4xl font-bold">
-									{formatScheduledTime(dep.scheduledDepartureTime)}
+					{#if getArrivalStatus(dep.predictedDepartureTime, dep.scheduledDepartureTime)}
+						{@const status = getArrivalStatus(
+							dep.predictedDepartureTime,
+							dep.scheduledDepartureTime
+						)}
+						<div class="flex items-center p-5">
```

Comment:
please extract this into a Svelte component and add a really basic unit test for it (i.e. just to make sure it renders successfully)